examples/turso-nextjs-starter/app/add-new/page.tsx

```diff
@@ -0,0 +1,92 @@
+import './form.css'
+
+export const runtime = 'edge'
+
+export const metadata = {
+  title: 'Add New',
+  description: 'Contribute to the frameworks list.',
+}
+
+export default function AddNewPage(request: { searchParams: any }) {
+  const { error, message } = request.searchParams
+
+  return (
+    <>
+      <h1>Submit a framework</h1>
+
+      <div className="mb-32 text-center lg:text-left w-[80vw] max-w-2xl flex flex-col">
+        {message && (
+          <div className="bg-green-200 text-green-800 p-2 w-full">
+            {message}
+          </div>
+        )}
+        {error && (
+          <div className="bg-red-200 text-red-800 p-2 w-full">{error}</div>
+        )}
+        <form
+          action="/api/add-framework"
```

Comment:
This should probably use server actions, rather than manually POSTing to this API.